### ISSUE TITLE
You can disable auto update by setting SUEnableAutomaticChecks

### ISSUE THREAD

**Issue by krackers** [qzz0518/remove-HyperSwitch-expired-dialog#6]
I believe you can disable auto update check by doing `defaults write com.bahoom.HyperSwitch SUEnableAutomaticChecks -bool NO`. The app always resets `SUScheduledCheckInterval`, but it doesn't seem to reset `SUEnableAutomaticChecks`.

This will get rid of periodic auto update, but you may still have the `expired` dialog show on system restart.

**Comment by qzz0518** [comment]
Very good idea!
After updating the latest version, I did not recive the error prompt dialog pop up. 
May I ask if there will be a new prompt dialog after you update to the latest version?

**Comment by krackers** [comment]
You are correct, after updating to the latest version there is no more prompt dialog. Hopefully the developer will continue to keep the app updated.

**Comment by tillt** [comment]
...and we all know that he won't update in time for some bizarre reasons.
